### PR TITLE
Update screenshot and documentation related to shared databases.

### PR DESCRIPTION
**Pull request by obraliar**
Changing PR: https://github.com/JabRef/jabref/pull/2009

### CODE REVIEW

File: en/SQLDatabase.md [hunk 1]
```diff
@@ -53,4 +54,4 @@ Otherwise you might override someone else's work.
 
 ## Try it out
 
-You can test the shared database support by using <https://www.freemysqlhosting.net>.
\ No newline at end of file
+You can test the shared database support by using <https://www.freemysqlhosting.net>.
```
Comment (mlep) [inline en/SQLDatabase.md]: Shared **SQL** database

Comment (Siedlerchr) [inline en/SQLDatabase.md]: I just edited this directly. I just merged the other PR in, did not see this one here.

File: en/SQLDatabase.md [hunk 2]
```diff
@@ -12,8 +12,8 @@ If you used to work with an older version of JabRef, please refer to [SQLDatabas
 ## Usage
 
 To use this feature you have to connect to a remote database. 
-To do so you have to open **File** in the menu bar and then click the **Open shared database** item.
-The **Open shared database** dialog will open and you will have to fill in the shared's database connection settings.
+To do so you have to open **File** in the menu bar and then click the **Connect to shared database** item.
```
Comment (Siedlerchr) [inline en/SQLDatabase.md]: @mlep  Best is to create a new issue in the JabRef repo as folllow up to 
JabRef/jabref#2009
becasuse the Screenshots would have to be changed, too

**Comment by Siedlerchr** [comment]
I merged the other one in and added shared SQL database in the last sentence. 
Therefore I close this one to avoid further confusion.
Further changes go in a new PR.

**Comment by koppor** [comment]
"The other one" probably is https://github.com/JabRef/help.jabref.org/pull/120

**Pull request closed without merging**